src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+
+  const communityAccessibilitySite = `${props.communityAccessibilityLink}`;
+  const communityAccessibilityElement = (
+    <a href={communityAccessibilitySite}>Website Accessibility Policy</a>
+  );
+
+  /* eslint-disable max-len */
```

Comment:
Out of curiosity, why disable the max-len rule? Does breaking the long text into shorter lines affect how it's displayed in the browser?

Reply:
It doesn't affect the display, but I didn't want to have to muck around with formatting the text essentially. So...disabling the line count is partially out of laziness, but also in that we get the copy from legal and I'd rather just copy/paste.